src/renderer/wg_engine/tvgWgRenderer.cpp

```diff
@@ -273,26 +275,99 @@ bool WgRenderer::target(void* window, uint32_t w, uint32_t h)
     mSwapChain = wgpuDeviceCreateSwapChain(mContext.device, mSurface, &swapChainDesc);
     assert(mSwapChain);
 
-    mRenderTarget.initialize(mContext, mPipelines, w, h);
+    mRenderTargetWnd.initialize(mContext, mPipelines, w, h);
+    mRenderTargetRoot.initialize(mContext, mPipelines, w, h);
     return true;
 }
 
 
 Compositor* WgRenderer::target(TVG_UNUSED const RenderRegion& region, TVG_UNUSED ColorSpace cs)
```

Comment:
Reffering composition region would help to reduce the composition area significantly.